destination/destination.go

```diff
@@ -49,6 +50,7 @@ type Destination struct {
 	SpoolSyncPeriod      time.Duration
 	SpoolSleep           time.Duration // how long to wait between stores to spool
 	UnspoolSleep         time.Duration // how long to wait between loads from spool
+	RouteName            string        // how long to wait between loads from spool
```

Comment:
this comment doesn't make any sense

Reply:
yeah, my bad, copied from line above, will remove that